packages/unplugin/src/types.ts

```diff
@@ -84,3 +88,12 @@ type IncludeEntry = {
   //TODO: what about the other entries??
 };
 */
+
+/**
+ * Holds data for Sentry internal purposes
+ * (e.g. telemetry and logging)
+ */
+export type SentryContext = {
```

Comment:
`s/SentryContext/BuildContext/`? Just to indicate that this can be used for more stuff than just Sentry SDK objects?